pkg/manifest/anaconda_installer_iso_tree.go

```diff
@@ -780,3 +782,15 @@ systemctl enable osbuild-subscription-register.service
 `
 	return fmt.Sprintf(kickstartSubscriptionPost, fullSourcePath, dest)
 }
+
+func makeKickstartInsightsClientBootPost() string {
+	// Enabling the insights-client.service is necessary for the insights client to
+	// collect when a system is rebooted after an upgrade.
+	// It is facilitated via insights-client.service.d/override.conf
+	kickstartInsightsClientBootPost := `
+%%post
+systemctl enable insights-client.service
+%%end
+`
+	return fmt.Sprintf(kickstartInsightsClientBootPost)
```

Comment:
```suggestion
	return fmt.Sprint(kickstartInsightsClientBootPost)
```

Reply:
Why are we even printing it? What change to the string we want to hapen through the print? :thinking:

Reply:
Well... now I feel stupid :sweat_smile: 

Yeah, just return the string.

Reply:
Yeah. That was a wee hours cut and paste. Can just return the string since it's one var.